pkg/controller/composite/metacontroller.go

```diff
@@ -85,12 +85,12 @@ func NewMetacontroller(controllerContext common.ControllerContext, mcClient mccl
 	return mc
 }
 
-func (mc *Metacontroller) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+func (mc *Metacontroller) Reconcile(context context.Context, request reconcile.Request) (reconcile.Result, error) {
```

Comment:
Is the variable here shadowing the `context` package name? Usually the variable is called `ctx` to avoid having the same name as the package.

Reply:
done